signs-work-order-map/config/map.ts

```diff
@@ -24,6 +24,7 @@ export const DEFAULT_MAP_PARAMS = {
   boxZoom: false,
   mapboxAccessToken: MAPBOX_TOKEN,
   maxBounds: MAP_MAX_BOUNDS,
+  maxZoom: 20,
```

Comment:
it zooms in very close when there is only one sign, so I added this. open to feedback!

Reply:
i feel like it still zooms in pretty close!! could be zoomed out even more imo

Reply:
Hmm yeah the initial map zoom on one sign is real close in. I assume technicians do need the ability to zoom way far in order to place sign locations correctly, or to click on the right location when they are very close together.